src/services/notificationService.ts

```diff
@@ -93,48 +93,80 @@ export default class NotificationService {
             );
           }
         );
-      });
-    });
+      }
+    );
 
   public readonly createOrUpdateInstallation = (
     fiscalCode: FiscalCode,
     installationID: InstallationID,
     installation: Installation
-  ): Promise<IResponseErrorInternal | IResponseSuccessJson<SuccessResponse>> =>
-    withCatchAsInternalError(() => {
-      const notificationHubService = azure.createNotificationHubService(
-        this.hubName,
-        this.endpointOrConnectionString
+  ): Promise<
+    IResponseErrorInternal | IResponseSuccessJson<SuccessResponse>
+  > => {
+    const azureInstallation: IInstallation = {
+      installationId: !this.allowMultipleSessions.allowMultipleSessions
```

Comment:
can you please add a comment on the reason of this logic here ?

Reply:
Fixed in aea043b